include/VaultClient.h

```diff
@@ -248,11 +250,14 @@ namespace Vault {
             }
 
             long responseCode = 0;
+            char *url = nullptr;
```

Comment:
```suggestion
            std::string url;
```

Reply:
No problem; I have provided an update for this and also updated the test suite.